client/my-sites/plan-features/index.jsx

```diff
@@ -272,6 +273,7 @@ class PlanFeatures extends Component {
 						basePlansPath={ basePlansPath }
 						relatedMonthlyPlan={ relatedMonthlyPlan }
 						isInSignup={ isInSignup }
+						selectedPlan={ selectedPlan }
```

Comment:
That won't be enough for mobile. I think you'll need to also add this to `<PlanFeaturesHeader />` in `renderMobileView()`.

Reply:
ofc! hat tip :)